README.md

```diff
@@ -56,17 +55,21 @@ Apart from the main parameters listed above, you should choose the command to ru
 ### Obtaining the GitHub personal access token
 Create your own Github access token (please refer to the instructions [here](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token)). When creating this token, make sure to assign **at least** `repo` permissions.
 
-The GitHub token should be stored in an environment variable and supplied to Macaron via command line parameters.
+The GitHub token should be stored in an **environment variable** called `GITHUB_TOKEN`. Macaron will read the value of this Github token from the environment variable **only** if we use the `analyze` command (see instructions below).
 
 ### Analyzing SLSA levels of a repository
 *This section describes the `analyze` command of Macaron.*
 
 Run this command to determine the SLSA level of a repository:
 
 ```bash
+export GITHUB_TOKEN="<your token here>"
+
 python -m macaron [-v] analyze -rp <path_to_target_repo> [-b <branch_name>] [-d <digest>] -c <config_path> [-sbom <sbom_path>]
 ```
 
+**Note**: for the rest this section, we assume that the Github access token has been set as the environment variable correctly.
```

Comment:
**Note**: for the rest of this section, we assume that the Github access token has been set as `GITHUB_TOKEN` environment variable.